README.md

```diff
@@ -175,16 +191,20 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 For more information about the app registration:
 
+- [Microsoft identity platform (Azure Active Directory) for developers](https://aka.ms/aaddevv2)
 - [Quickstart: Register an application with the Microsoft identity platform (Preview)](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app)
 - [Quickstart: Configure a client application to access web APIs (Preview)](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-access-web-apis)
 
 For more information, see MSAL.NET's conceptual documentation:
 
+- [MSAL.NET's conceptual documentation](https://aka.ms/msal-net)
+- [Customizing Token cache serialization](https://aka.ms/msal-net-token-cache-serialization)
+- [Scenarios](https://aka.ms/msal-net-scenarios)
+- [Acquiring Tokens](https://aka.ms/msal-net-acquiring-tokens)
 - [Username/password](https://aka.ms/msal-net-up)
-- [Customizing Token cache serialization](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/token-cache-serialization) (was not done in this sample, but you might want to add a serialized cache)
 - [Azure Active Directory v2.0 and the OAuth 2.0 resource owner password credential](https://review.docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) to learn more about the underlying protocol
 - [Resource owner password credentials RFC](https://tools.ietf.org/html/rfc6749#section-1.3.3)
 
-For more information about the Azure AD v2.0 endpoint see:
+For more information about the Microsoft identity platform (AAD) v2.0 see:
```

Comment:
>v2.0 [](start = 65, length = 4)

the v2.0 needs not be appended when using "Microsoft identity platform (AAD)" .

Reply:
I'm surprised @kalyankrishna1 : AG  told me that ….

Reply:
Please follow AG's advice Jean-Marc. I might have misunderstood

---
In reply to: [257620524](https://github.com/Azure-Samples/active-directory-dotnetcore-console-up-v2/pull/8#discussion_r257620524) [](ancestors = 257620524)